documentation/openapi.json

```diff
@@ -49,9 +49,7 @@
             "description": "Process ID used to identify the target process.",
             "schema": {
               "type": "integer",
-              "description": "Process ID used to identify the target process.",
```

Comment:
My educated guess here is that: (1) the description doesn't need to be duplicated in the schema since it is already on the query parameter, and (2) marking query parameters as nullable is unnecessary since query parameters are optional when specifying a URL.